next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationsHeader.tsx

```diff
@@ -30,23 +31,22 @@ const MyApplicationsHeader = (props: MyApplicationsHeaderBase) => {
         <ul className="flex gap-0 lg:gap-12">
           {headerNavigationList.map((item, i) => (
             <li className="w-full lg:w-max" key={i}>
-              <button
+              <Link
+                href={`#${item.tag}`}
                 type="button"
```

Comment:
Shouldn't we remove
`type="button"`?

Maybe it's appropriate tu use MLinkNew (don't know)?

Reply:
Yes, we should remove it. I think there is no need for `type` in the Link component.

However I am refactoring this PR to use query params for navigations instead of # in the URL and I am using `<button>` tag again instead of `Link` component. But if I end up with `Link` component I will remove the `type="button"` attribute.

I am not familiar with MLinkNew, is it a component?

Reply:
I think this should be superseded by refactoring to react-aria tabs as the comment above suggests https://react-spectrum.adobe.com/react-aria/Tabs.html

perhaps a good followup ?